packages/editor/src/components/post-saved-state/test/__snapshots__/index.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`PostSavedState returns a disabled button if the post is not saveable 1`
 <button
   aria-disabled="true"
   aria-label="Save draft"
-  class="components-button has-text has-icon"
+  class="components-button has-icon"
```

Comment:
Thanks to the failure of this unit test I have found another problem.

When the screen width is narrow, only an icon is shown on the `Button` that indicates the save status. The `has-text` class should not be given, but in the trunk, The `has-text` class is shown for a certain status, which causes strange behaviour.

https://user-images.githubusercontent.com/54422211/190667587-c1cdf0e2-eeb7-4b60-88e5-32801a989bc5.mp4

This PR will solve this problem as well. Therefore, this snapshot update is intended.

https://user-images.githubusercontent.com/54422211/190667617-0b4185b5-3d20-4d68-abf9-9aedee3ddb8c.mp4